meilisearch/src/option.rs

```diff
@@ -769,41 +767,48 @@ impl Deref for MaxThreads {
     }
 }
 
-fn load_certs(filename: PathBuf) -> anyhow::Result<Vec<rustls::Certificate>> {
+fn load_certs(
+    filename: PathBuf,
+) -> anyhow::Result<Vec<rustls::pki_types::CertificateDer<'static>>> {
     let certfile =
         fs::File::open(filename).map_err(|_| anyhow::anyhow!("cannot open certificate file"))?;
     let mut reader = BufReader::new(certfile);
     certs(&mut reader)
-        .map(|certs| certs.into_iter().map(rustls::Certificate).collect())
+        .collect::<Result<Vec<_>, _>>()
         .map_err(|_| anyhow::anyhow!("cannot read certificate file"))
 }
 
-fn load_private_key(filename: PathBuf) -> anyhow::Result<rustls::PrivateKey> {
+fn load_private_key(
```

Comment:
I think you can probably use `rustls_pemfile::private_key()` instead of `rsa_private_keys()` and `pkcs8_private_keys()`.

Reply:
Huuum, I don't think it's equivalent because currently, we read both the rsa and pkcs8 and then pick the first pkcs8 if there is one.
`private_key` would return the first key, whatever it is, so we may pick an RSA key where we would choose the pkcs8 key in the past.

I’m not confident enough to make this change tbh 
I don't know what it could break and if people are relying on it